tensorboard/webapp/widgets/histogram/histogram_v2_component.scss

```diff
@@ -125,15 +129,19 @@ svg {
     stroke-dasharray: 2;
   }
 
+  $_path-circle-opacity: 0.6;
```

Comment:
Why increase the opacity from .5 to .6? Not an objection but just in case this is a typo.

Reply:
Yes, it was intentional. With some improvement here, it did not look ideal so I had bumped up the number by 0.1.